index.html

```diff
@@ -290,10 +290,19 @@
 //   Output: The object to be sent, or something falsy if no
 //           object should be sent
 var localCommands = {
-  flush: function() {
+  clear: function() {
     delete localStorage.history;
     box.innerHTML = "";
-  }
+  },
+  help: function() {
+    var cmds = Object.keys(localCommands).concat(Object.keys(commands));
+    cmds.sort();
+    box.innerHTML = box.innerHTML +
+      '<br/>commands available: <br/>&nbsp;&nbsp;/' +
+      cmds.join('<br/>&nbsp;&nbsp;/');
+    box.scrollTop = box.scrollHeight - 10;
+    return false;
```

Comment:
Ack.  Will fix in my next PR.  I had this in commands instead of localCommands, before I made the latter work.